canvas_cli/tests.py

```diff
@@ -23,11 +24,16 @@ def plugin_name() -> str:
     return f"cli-{datetime.now().timestamp()}".replace(".", "")
 
 
-@pytest.fixture(autouse=True, scope="session")
-def create_or_update_config_auth_file_for_testing(plugin_name: str) -> Generator[Any, Any, Any]:
+@pytest.fixture(scope="session")
```

Comment:
@mbiannaccone I has to remove  `autouse=True` from this fixture to prevent it from executing when running only unit tests.